packages/block-template/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "block-template",
-  "version": "0.0.18",
+  "version": "0.0.17-beta",
```

`0.16` is the latest published version

---

```suggestion
  "version": "0.0.17-canary.0",
```

Beta, alpha or RC imply certain levels of readiness. We don’t have a classification of these (yet), so it’s better to avoid words that carry extra meaning. `canary` (or `next`) just mean work in progress. 

Adding numbers is also common, so we can stick to using them too.

---

I have updated this, and will update other packages as I publish new versions (rather than publishing identical versions of everything now to give them `canary` tags).